manifests/cluster-authentication-operator_02_config.crd.yaml

```diff
@@ -5,10 +5,71 @@ metadata:
 spec:
   scope: Cluster
   group: operator.openshift.io
-  version: v1
   names:
     kind: Authentication
     plural: authentications
     singular: authentication
-  subresources:
-    status: {}
+  versions:
+    - name: v1
+      served: true
+      storage: true
+  validation:
+    # Ensure we will be able to deserialize the object into the golang type
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          required: ["managementState"]
+          properties:
+            managementState:
+              enum:
+                - "Force"
+                - "Managed"
+                - "Unmanaged"
+                - "Removed"
+              default: "Managed"
+            logLevel:
+              enum:
+                - "Normal"
+                - "Dubug"
+                - "Trace"
+                - "TraceAll"
+            operandSpecs:
+              type: array
+              items:
+                type: object
+                required: ["name", "operandContainerSpecs", "unsupportedResourcePatches"]
+                properties:
+                  name:
+                    type: string
+                  operandContainerSpecs:
+                    type: array
+                    items:
+                      type: object
+                      required: ["name"]
+                      properties:
+                        name:
+                          type: string
+                        resources:
+                          type: object
+                          properties:
+                            limits:
+                              type: object
+                            requests:
+                              type: object
+                  unsupportedResourcePatches:
+                    type: array
+                    items:
+                      type: object
+                      required: ["type", "patch"]
+                      properties:
+                        type:
+                          type: string
+                        patch:
+                          type: string
+            unsupportedConfigOverrides:
+              type: object
+            observedConfig:
+              type: object
```

Comment:
If we leave out unsupportedConfigOverrides and observedConfig, that means nobody can set them, right?  So, we need to list all possible properties?

Reply:
^^ seems I'm wrong about that: https://github.com/kubernetes/kubernetes/issues/59151  I should proably _not_ include these, since we're not actually able to validate past 'object'